sdk/core/azure-core/src/test/java/com/azure/core/util/polling/PollerTests.java

```diff
@@ -856,7 +856,7 @@ public void waitUntilOperationWithTimeout() {
             if (invocationCount[0] == 0) {
                 return Mono.just(expected);
             } else {
-                return Mono.delay(Duration.ofSeconds(2))
+                return Mono.delay(Duration.ofSeconds(5))
```

Comment:
Are these tests making service calls? If not, should we use virtual timer to speed this up?

Reply:
It isn't making a service call but it's a scenario where we want time to actually block. This is testing a case where we set the waitUntil timeout to a few 100ms and never reach the state we wanted.